### DISCUSSION TITLE
Why regex is triggered with minor but not with patch

### DISCUSSION THREAD

**Discussion by mtardy** [renovatebot/renovate#23453] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.13.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I would like the patch updates to also update the regex-matched versions.

For some reason, it works with the minor bump but not the patch. It's not detecting it should do the regex as well: https://github.com/mtardy/tetragon/pulls
- minor is fine https://github.com/mtardy/tetragon/pull/63
- patch ignores the regex match https://github.com/mtardy/tetragon/pull/64
- here's the run with the debug log https://github.com/mtardy/tetragon/actions/runs/5598423045/jobs/10238040036#step:6:71
- the config https://github.com/mtardy/tetragon/blob/main/.github/renovate.json5

### What you tried so far.

I have no longer good idea on what to do, browsing the debug log it's not obvious and I just see that this update seems to not take the regex update while the other is doing fine.

### Relevant debug logs

<details><summary>Logs</summary>

https://github.com/mtardy/tetragon/actions/runs/5598423045/jobs/10238040036#step:6:71

</details>

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#separateminorpatch

**Reply by mtardy** [reply]
Yes indeed I activated that, I'm happy that it's generating two PRs, my issue is that one lacks the regex update :/.

**Reply by mtardy** [reply]
Indeed I only got one update for that dependency https://github.com/mtardy/tetragon/actions/runs/5598423045/jobs/10238040036#step:6:5611 while I got two for the other one, https://github.com/mtardy/tetragon/actions/runs/5598423045/jobs/10238040036#step:6:1419.

Since I put the `separateMinorPatch` on the top level, why does it not behave similarly for the regexes?

**Reply by mtardy** [reply]
oh my god I just forgot I added that and that was picked up, it was intended only at the github-actions manager https://github.com/mtardy/tetragon/blob/main/.github/renovate.json5#L74-L79, I need to match the managers as well... Thank you so much you really helped me narrow that one, debug logs are so large I didn't know where to look.
